core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/WorkflowExecutor.scala

```diff
@@ -0,0 +1,93 @@
+package edu.uci.ics.amber.engine.architecture.scheduling
+
+import com.twitter.util.Future
+import com.typesafe.scalalogging.LazyLogging
+import edu.uci.ics.amber.engine.architecture.common.AkkaActorService
+import edu.uci.ics.amber.engine.architecture.controller.{ControllerConfig, ExecutionState}
+import edu.uci.ics.amber.engine.common.rpc.AsyncRPCClient
+
+import scala.collection.mutable
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+class WorkflowExecutor(
```

Comment:
rename to ExecutionCoordinator?

Reply:
renamed to `WorkflowExecutionController`